cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -201,20 +203,34 @@ runQueryTip
 runQueryTip (AnyConsensusModeParams cModeParams) network mOutFile = do
   SocketPath sockPath <- firstExceptT ShelleyQueryCmdEnvVarSocketErr readEnvSocketPath
   let localNodeConnInfo = LocalNodeConnectInfo cModeParams network sockPath
-      consensusMode = consensusModeOnly cModeParams
+  let consensusMode = consensusModeOnly cModeParams
 
   anyEra <- determineEra cModeParams localNodeConnInfo
-  tip <- liftIO $ getLocalChainTip localNodeConnInfo
-
-  let tipSlotNo = case tip of
-        ChainTipAtGenesis -> 0
-        ChainTip slotNo _ _ -> slotNo
 
   case consensusMode of
     CardanoMode -> do
-        eResult <- liftIO $ queryEraHistoryAndSystemStart localNodeConnInfo Nothing
-        case eResult of
-          Left acqFail -> left (ShelleyQueryCmdAcquireFailure acqFail)
+        (chainTip, result) <- liftIO $ queryChainTipAndEraHistoryAndSystemStart localNodeConnInfo Nothing
+
+        let tipSlotNo = case chainTip of
+              ChainTipAtGenesis -> 0
+              ChainTip slotNo _ _ -> slotNo
+
+        case result of
+          Left acqFail -> do
+            let output = encodePretty $ O.QueryTipOutput
```

Comment:
Do we really need to qualify this? `Cardano.CLI.Shelley.Output` lives in the same library.